### PR TITLE
Feat/change to intellyo visual regression

### PR DESCRIPTION
**Pull request by molnosikristof**

### CODE REVIEW

**Comment by molnosikristof** [comment]
The tooltip test fails will disappear when we marge it to master a ccording to Kristof.

File: wdio.conf.js [hunk 1]
```diff
@@ -155,7 +158,12 @@ exports.config = {
   sauceConnect: true,
 
   sauceConnectOpts: {
-    tunnelIdentifier: os.hostname()
+    tunnelIdentifier: os.hostname(),
+    logfile: './sauce-connect.log',
```
Comment (oroce) [inline wdio.conf.js]: should we keep this?

File: wdio.conf.js [hunk 2]
```diff
@@ -169,7 +177,7 @@ exports.config = {
 
   mochaOpts: {
     ui: 'bdd',
-    timeout: 99999999,
+    timeout: 60000,
```
Comment (oroce) [inline wdio.conf.js]: will this be enough? I'm just curious.

File: wdio.conf.js [hunk 3]
```diff
@@ -65,7 +66,9 @@ if (e2eProfile.includes('sauceextended')) {
   driver = 'sauce';
   sauceLabsUsername = process.env.SAUCE_LABS_USERNAME;
   saucelabsAccesKey = process.env.SAUCE_LABS_ACCESS_KEY;
-
+  if (e2eProfile.includes('nightly')) {
```
Comment (oroce) [inline wdio.conf.js]: why do you need this? Why don't you set the `MAX_INSTANCE` env variable in `.circle/config.yml`'s nightly workflow?

File: wdio.conf.js (hunk 2 above)
Comment (molnosikristof) [reply wdio.conf.js]: Typically yes, but we can increase this to 100000

File: wdio.conf.js (hunk 1 above)
Comment (molnosikristof) [reply wdio.conf.js]: No, we shouldn't, but we can use the provided infos when its necessary. Maybe it's overkill a bit.

File: .circleci/config.yml [hunk 4]
```diff
@@ -43,7 +43,7 @@ jobs:
 
       - run:
           name: test
-          command: npm run test
+          command: MAX_INSTANCE=1 E2EPROFILE=sauceextended npm run test
```
Comment (oroce) [inline .circleci/config.yml]: actually if you want this to have in a more readable version you can do:

```
- run:
    name: test
    command: npm run test
    environment:
       MAX_INSTANCE: 1
       E2EPROFILE: sauceextended
```

**Pull request closed without merging**